### ISSUE TITLE
Update citation/publish pipeline

### ISSUE THREAD

**Issue by thorellk** [ctmrbio/BACTpipe#137]
With the 3.0 release we should update the Zenodo citation so that the correct people are acknowledged. Should we also try to publish the pipeline as some kind of short communication? Or is that to aim too high?

**Comment by abhi18av** [comment]
I think that this makes sense. While working on my masters, I realized that referencing book/articles is much easier than referencing source code and repositories. When I came across this pipeline I loved this idea and release my own work via zenodo https://zenodo.org/record/4294795 

I do think that academically speaking an updated citation for the latest stable version makes things more authentic.

**Comment by abhi18av** [comment]
And I'm also in favor of following this up with a short communication as well.

**Comment by boulund** [comment]
I can't remember exactly since it was a while since I set it up, but I think that if we make a release here on github it will be very easy to assign a new DOI using that release.

**Comment by abhi18av** [comment]
That makes sense, it's almost like a CI/CD. Btw, I'm curious does it parse the commits for list of people who have contributed?

![image](https://user-images.githubusercontent.com/12799326/104725123-a2cc4780-5757-11eb-9afc-f86fa242a76c.png)

**Comment by boulund** [comment]
Yes, I think it does

**Comment by boulund** [comment]
When you've tested that everything works for the 3.0 release in the main branch and all the docs have hade their final touches, can you make a release on github so I can link it into Zenodo?

**Comment by emilio-r** [comment]
Was there ever any resolution to this? It seems that the docs have been updated already. I am asking as I will be starting to write my thesis very soon and having a DOI then, by which to cite the work with BACTpipe 3.0, would be great.

**Comment by boulund** [comment]
Someone just needs to make a new release on Github so we can attach a new Zenodo DOI to that release.

**Comment by boulund** [comment]
Once we merge the pending changes from the develop branch into the main branch I'll make a new release and update the Zenodo stuff

**Comment by thorellk** [comment]
Hooray 😄 

Now when we're happily done with 3.1 maybe it is time to update the citation?

**Comment by boulund** [comment]
https://zenodo.org/record/4742358#.YJUyvbX7Q-U

**Comment by boulund** [comment]
I think we missed the citation in the docs,
https://bactpipe.readthedocs.io/en/latest/#citing-bactpipe

can someone make a commit fixing that?

**Comment by thorellk** [comment]
Hmm, seems like Sandra is left out and the naming scheme is quite weird, mixing GitHub user id's and full names, including capitalized ones. I guess this is collected from the GitHub user information? Is there any way of editing this?

**Comment by boulund** [comment]
Yes, it automatically collects the names/usernames of everyone who has contributed commits. We can edit it manually if we want to. We just need to decide which authors should be there and in which order.

**Comment by thorellk** [comment]
Emilio Rudbeck; Abhinav Sharma; Yue Hu; Joseph Kirangwa; Emilyn Costa Conceição; Sandra Álvarez-Carretero; Fredrik Boulund; Kaisa Thorell? Or if you want to be the last?

**Comment by boulund** [comment]
That looks fine with me, I'll update the citation on Zenodo and in the docs right away.

**Comment by boulund** [comment]
I have updated the Zenodo citation and the citation on the first page of the docs, and synced the little commit from master to develop, so we can continue with feature branches from develop for future developments.

**Comment by thorellk** [comment]
@emilio-r and @abhi18av, were you interested in writing something on the pipeline as well to publish it as some kind of paper?

**Comment by thorellk** [comment]
@boulund you forgot Sandra! She was the original author of BACTpipe and even if much has changed since I still think she should be included.

**Comment by boulund** [comment]
I don't understand what you mean, Sandra is included? 
![image](https://user-images.githubusercontent.com/4919402/117540918-b7d42b80-b011-11eb-8549-336da3e3009f.png)

**Comment by abhi18av** [comment]
> @emilio-r and @abhi18av, were you interested in writing something on the pipeline as well to publish it as some kind of paper? 

Yup, that'd be great. Perhaps a short publication explaining the workflow and the chioces behind this?

**Comment by thorellk** [comment]
Sandra was missing in the docs, not in the Zenodo doi. Sorry for not being clear, I added it now :)

**Comment by emilio-r** [comment]
First: Huzzah to having released 3.1!  :tada: 

> @emilio-r and @abhi18av, were you interested in writing something on the pipeline as well to publish it as some kind of paper?

Yes, writing something would be a good way to cap this whole experience. Next week I may find some time as my thesis will be in for review. Then I could probably write together a shorter report on BACTpipe to share for general review and comments, unless anyone else feels like writing, or have another suggestion for how to do this.
